platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ArtifactSelectionIntegrationTest.groovy

```diff
@@ -1170,31 +1169,6 @@ class ArtifactSelectionIntegrationTest extends AbstractHttpDependencyResolutionT
             }
         """
 
-        file("ui/build.gradle") << """
-            $header
-            def attr = Attribute.of('attr', Number)
```

Comment:
💭 This project is now successfully found, as Number can be coerced to String when the local metadata is created.